packages/services/api/src/modules/audit-logs/providers/audit-logs-manager.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable, Scope } from 'graphql-modules';
+import { ClickHouse, sql } from '../../operations/providers/clickhouse-client';
+import { Logger } from '../../shared/providers/logger';
+import { AuditLogEvent, auditLogSchema } from './audit-logs-types';
+
+// TODO: Error: No provider for ClickHouse! (AuditLogManager -> ClickHouse) - in Module "audit-logs" (Operation Scope)
+@Injectable({
+  scope: Scope.Operation,
+  global: true,
+})
+export class AuditLogManager {
+  private logger: Logger;
+
+  constructor(
+    logger: Logger,
+    private clickHouse: ClickHouse,
+  ) {
+    this.logger = logger.child({ source: 'AuditLogsManager' });
+  }
+
+  async createLogAuditEvent(event: AuditLogEvent): Promise<void> {
+    const { eventTime, eventType, id, organizationId, user } = event;
```

Comment:
instead of taking the date as input. create the `new Date` in this function and let it control. I don't see why we would want the consumer to set date always.

Reply:
Fixed! thanks